serverless.yml

```diff
@@ -27,6 +27,16 @@ provider:
       - Fn::Sub: ${SoclessLambdaVpcSG.GroupId}
     subnetIds:
       - Ref: SoclessPrivateFunctionSubnet
+  stackPolicy:
+    - Effect: Allow
+      NotAction: Update:Delete
+      Principal: "*"
+      Resource : "*"
+    - Effect: Deny
+      Action: Update:Replace
+      Principal : "*"
+      Resource: "SoclessEIP"
```

Comment:
Is this a static string or the resource logical ID? If its not a static string and we can use the resource logical ID instead let's do that.

Reply:
updated with correct policy